### PR TITLE
add python10 in line 4

### PR DESCRIPTION
**Pull request by smalllun**
给老子merge！

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## build codespace for languages
 
 ### java 9
-
+### python 10
```
Comment (Alice52) [inline README.md:4]: python 10 is deprecate! @smalllun

**Review by Alice52 — COMMENTED** [review]
python 10 is deprecate! @smalllun

File: README.md (hunk 1 above)
Comment (smalllun) [reply README.md:4]: 我不服

Comment (Alice52) [reply README.md:4]: 同意

**Pull request merged**